.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
+updates:
+- package-ecosystem: pip
+  directory: "/"
+  schedule:
+    interval: daily
+    time: "13:00"
+  open-pull-requests-limit: 10
+  ignore:
+  - dependency-name: nr-databind-core
```

Comment:
Why is this needed?

Reply:
The whole of dependabot or just nr-databind-core? I used the same config from allennlp to keep things consistent.

Reply:
Ah, gotcha. We definitely want dependabot, so thanks for adding that. But I don't think we need this `ignore` section for `nr-databind-core`.